src/jsMain/js/simulator/windows/SimulatorSettingsWindow/SimulatorSettingsWindow.scss

```diff
@@ -0,0 +1,141 @@
+:global {
```

Comment:
I had no idea what `:global` did, so in case anyone else is curious:

https://github.com/webpack-contrib/css-loader#features
https://github.com/css-modules/css-modules#exceptions